src/tests/EdFi.DataManagementService.Tests.E2E/Features/InvalidReferenceValidation.feature.disabled

```diff
@@ -0,0 +1,125 @@
+Feature: Invalid Reference Validation
+    POST requests validating invalid references
+
+    //Scenario needs to work with 409
+        @ignore
+        Scenario: PENDING Ensure clients cannot create a resource with an invalid reference
+             When a POST request is made to "ed-fi/studentEducationOrganizationAssociations" with
+                  """
+                  {
+                      "educationOrganizationReference": {
+                          "educationOrganizationId": 19255901
+                      },
+                      "studentReference": {
+                          "studentUniqueId": "1000080"
+                      },
+                      "studentIndicators": [
+                          {
+                              "indicatorName": "Fake",
+                              "indicator": "Fake"
+                          }
+                      ]
+                  }
+                  """
+             Then it should respond with 403
```

Comment:
@elopezgap @msilesgap this is not the correct response, so we need another pull request. Yes, this is the correct response from the ODS/API, but it is not the correct response for the Data Management Service.

Use the next scenario to find the expected response. This is what I was trying to explain on Thursday, Esteban, using Course as an example.